src/Microsoft.TestPlatform.ObjectModel/Navigation/FullSymbolReader.cs

```diff
@@ -138,14 +173,29 @@ private DiaNavigationData GetSymbolNavigationData(IDiaSymbol symbol)
 
             try
             {
-                this.session.findLinesByAddr(symbol.addressSection, symbol.addressOffset, (uint)symbol.length, out lines);
+                // Get the address section
```

Comment:
Should we combined all three 'if' statements with `||`.

Reply:
we can, but I think code will be less readable.